mount_darwin.go

```diff
@@ -27,6 +27,14 @@ func loadOSXFUSE(bin string) error {
 	return err
 }
 
+func loadMacFuseIfNeeded(devPrefix string, bin string) error {
+	if _, err := os.Stat(devPrefix + "0"); os.IsNotExist(err) {
```

Comment:
Do we not need to loop the suffixes anymore like the original code does?

Reply:
No this is just to make sure the device file exists here but don't actually try to open it (that's the job of the mounter binary we are calling in exec). Since it starts with 0 checking zero should be enough.

Reply:
I added a comment here